index.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Kinto React Boilerplate</title>
-   <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+  <title>I really hate money</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
I agree, accessibility is important. I believe this rule was added to support some specific display (phone related).